src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -141,7 +141,7 @@ private void initializeProducer() throws Exception {
     producerProps.putAll(_producerPropsOverride);
 
     _producer = (KMBaseProducer) Class.forName(_producerClassName).getConstructor(Properties.class).newInstance(producerProps);
-    LOG.info("Producer is initialized.");
+    LOG.info(_name + ": Producer is initialized.");
```

Comment:
There might be a better way to configure the loggers to add what we need to them.  I would rather explore that in a different PR.